src/bots/discord/utils/praiseEmbeds.ts

```diff
@@ -0,0 +1,18 @@
+import { MessageEmbed } from 'discord.js';
+
+export const praiseErrorEmbed = (title: string, description: string) => {
+  return new MessageEmbed()
+    .setColor('#ff0000')
+    .setTitle(`ERROR: ${title}`)
+    .setDescription(description)
+    .setFooter({ text: 'PRAISE DID NOT REGISTER' });
+};
+
+export const praiseSuccessEmbed = (praised: string[]) => {
```

Comment:
I think we should make this message simpler and more like the one the current bot is using.

```
🥔 Vyvy-vi used /praise
Praise [BOT] today at 3:04 AM
@kristoferlund @someotherperson for this and that and a bunch of other stuff.
✅
```